physionet-django/console/views.py

```diff
@@ -946,6 +959,7 @@ def news_console(request):
     List of news items
     """
     news_items = News.objects.all().order_by('-publish_datetime')
+    news_items = paginate(request, news_items, 100)
```

Comment:
I'd just reduce this pages to 50 items, it was a little bit to much scrolling to get to the bottom (for me).

Reply:
Done